test/optimus/pr/test_inputs.json

```diff
@@ -0,0 +1,23 @@
+{
+  "TestOptimusPR.expected_bam_hash": "8c03262b5bd383abd237e5c1dab202bd",
+  "TestOptimusPR.expected_matrix_hash": "69f3be6085e0c5f694b3cfa877b0eeaa",
+  "TestOptimusPR.expected_matrix_summary_hash": "dd513351d4e7688c97f7bf902ba2876f",
+  "TestOptimusPR.expected_picard_metrics_hash": "7b7be5c9a2236920ca09f05811dca6d5",
+  "TestOptimusPR.fastq_inputs": [
+    [
+      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
```

Comment:
Do we have to worry about what projects things live in for our tests or if it should be a public place or anything like that? I have no context just throwing it out there, feel free to ignore.  Same for the other inputs.json

Reply:
We probably do eventually, but this is a systematic problem. Maybe @kbergin wants to make some tickets about this? I don't know when it was last discussed.

Reply:
I agree this is a larger discussion I think we should tackle when we have the whole team back together next Q. I think for now, getting some infrastructure set up and having automated tests is a great step forward.